WinUIGallery/Pages/SettingsPage.xaml.cs

```diff
@@ -39,6 +39,9 @@ public SettingsPage()
     {
         this.InitializeComponent();
         Loaded += OnSettingsPageLoaded;
+        ClearVisitedSamplesCard.IsEnabled = StringListSettingsHelper.GetList(SettingsKeys.RecentlyVisited).Count > 0;
+        UnfavoriteSamplesCard.IsEnabled = StringListSettingsHelper.GetList(SettingsKeys.Favorites).Count > 0;
+        SamplesSettingsExpander.IsExpanded = StringListSettingsHelper.GetList(SettingsKeys.RecentlyVisited).Count > 0 || StringListSettingsHelper.GetList(SettingsKeys.Favorites).Count > 0;
```

Comment:
Avoid loading data again, use ClearVisitedSamples.IsEnabled and UnfavoriteSamplesCard.IsEnabled instead. Ideally make it a computed property and reuse in the two click handlers below.